2-mandatory/4-restaurant.js

```diff
@@ -20,8 +20,23 @@ const MENU = {
 };
 
 let cashRegister = {
-  // write code here
-}
+  orderBurger(balance) {
```

Comment:
looks good!

it might be a bit over the top + unnecessary but because these two functions are basically the same, you could make a function which returns those functions 

<img width="613" alt="Screenshot 2023-04-01 at 16 34 19" src="https://user-images.githubusercontent.com/104007410/229298768-d9f54bd5-4875-4d31-8372-7bd9ce85f486.png">

then you could do 
<img width="577" alt="Screenshot 2023-04-01 at 16 35 30" src="https://user-images.githubusercontent.com/104007410/229298852-17fbc891-cb40-4bed-a9c5-33386e7bd3f2.png">